index.html

```diff
@@ -1809,18 +1824,19 @@ <h2>Hypermedia Controls</h2>
                 For humans, the information is usually text or images describing or decorating a hyperlink.
                 The control is a Web link, which includes at least the URI of the target resource,
                 which can be dereferenced by the Web browser (i.e., the link can be followed).
-                But also machines can follow links in a meaningful way, when the Web link is further described by a relation type and a set of target attributes.
-                A hypermedia control is the machine-understandable serialization of <em>how</em> to activate an affordance.
+                But also machines can follow links in a meaningful way, when the Web link is further described 
+                by a relation type and a set of target attributes.
+                A hypermedia control is the machine-understandable description of <em>how</em> to use an affordance.
                 Hypermedia controls usually originate from a Web server and are discovered in-band while a Web client is interacting with the server.
                 This way, Web servers can drive clients through Web applications dynamically,
                 by taking their current state and other factors such as authorization into account.
                 This is opposed to out-of-band interface descriptions that need to be preinstalled or hardcoded into clients
-                (e.g., RPC, WS-* Web services, HTTP services with fixed URI-method-response definitions).
+                (e.g., RPC, WS-* Web services, HTTP+REST services with fixed URI-method-response definitions).
```

Comment:
Please remove REST. There is no REST when the URIs are hard-coded and resources are typed through out of band information.

I know that too many people casually  call anything HTTP based REST(ful), but it is wrong.

Reply:
ok